### ISSUE TITLE
Modal aria-hidden elements should not be focussable Accessisbility Issue

### ISSUE THREAD

**Issue by qazizaahirah** [OfficeDev/office-ui-fabric-react#9642]
## Actual behavior:
When running the fast pass on a page with the modal popup opened, we hit the 'aria-elements should not be focusable' error.
Title: WCAG 4.1.2: ARIA hidden element must not contain focusable elements (div[data-is-visible="true"][aria-hidden="true"]:nth-child(1))
Tags: Accessibility, WCAG 4.1.2, aria-hidden-focus

Issue: ARIA hidden element must not contain focusable elements (aria-hidden-focus: https://dequeuniversity.com/rules/axe/3.2/aria-hidden-focus?application=msftAI)

Element path: div[data-is-visible="true"][aria-hidden="true"]:nth-child(1)

Snippet:

How to fix:
Fix all of the following:
Focusable content should have tabindex='-1' or be removed from the DOM

Despite applying the fix, we see the error repeatedly. When the modal is closed the error vanishes.

Environment:
Chrome version 74.0.3729.169

This accessibility issue was found using Accessibility Insights for Web 2.4.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

## Expected behavior:
This error is invalid as we are not able to focus any elements outside the modal (which is expected). We are also not able to tab-out of the modal unless intentionally by pressing Enter on the 'X' modal button - which is expected and the right behavior. The error seems misplaced.

## Documentation describing expected behavior
This error is invalid as we are not able to focus any elements outside the modal (which is expected). We are also not able to tab-out of the modal unless intentionally by pressing Enter on the 'X' modal button - which is expected and the right behavior. The error seems misplaced.

![image](https://user-images.githubusercontent.com/15657156/60456950-9f15b600-9bef-11e9-92a7-3ea4dd3269fa.png)

**Comment by khmakoto** [comment]
Hi @qazizaahirah, thanks for filing the issue! I'm not currently seeing this at least in our example page. I'll try to replicate this in a codepen to see how we move forward with this.

**Comment by cnamal** [comment]
We used to have this issue (#8697) and it was solved by #9019.
Hope this helps

**Comment by khmakoto** [comment]
This seems to be a duplicate of #9412 and it appears that is already being tracked there. I'll close this for the time being. Thanks!
